ui/admin/app/controllers/scopes/scope/host-catalogs/index.js

```diff
@@ -63,6 +64,10 @@ export default class ScopesScopeHostCatalogsIndexController extends Controller {
     isNew ? 'notifications.create-success' : 'notifications.save-success',
   )
   async save(hostCatalog) {
+    // If the role_arn is empty, then the credential type should be static
```

Comment:
Asking for learning purposes: If role_arn empty, means credential type should be static, shouldn't be doing this at the model or normlaizer? Or what is the reason for doing this within the controller?

Reply:
this is only for the UI, on page refresh this works fine as this gets correctly set in the [model](https://github.com/hashicorp/boundary-ui/blob/main/addons/api/addon/models/host-catalog.js#L54)